[1.4.x] Build error: call of overloaded 'abs(synfig::Real&)' is ambiguous

I've got the following error when building packages from [v1.4.x branch](https://github.com/synfig/synfig/commits/v1.4.x) (commit id 71775e696d32388b0f04a267c02a3785732f4bdc):

```
plant.cpp: In member function 'void Plant::sync() const':
plant.cpp:244:21: error: call of overloaded 'abs(synfig::Real&)' is ambiguous
  Real step(abs(step_));
                     ^
```

--- a/synfig-core/src/modules/mod_particle/plant.cpp
+++ b/synfig-core/src/modules/mod_particle/plant.cpp
@@ -241,7 +241,7 @@ Plant::sync()const
 
 	etl::hermite<Vector> curve;
 
-	Real step(abs(step_));
+	Real step(std::fabs(step_));
 
 	int seg(0);
 
@@ -582,7 +582,7 @@ Plant::draw_particles(Surface *dest_surface, const RendDesc &renddesc)const
 		std::vector<Particle>::iterator iter;
 		Particle *particle;
 		
-		float radius(size*sqrt(1.0f/(abs(pw)*abs(ph))));
+		float radius(size*sqrt(1.0f/(std::fabs(pw)*std::fabs(ph))));
 		
 		int x1,y1,x2,y2;
 		

